### DISCUSSION TITLE
Dokan support for windows server 2022

### DISCUSSION THREAD

**Discussion by subramanya-n** [dokan-dev/dokany#1152] [General]
Hi ,
       Do we have official support for Dokan on windows server 2022(Microsoft Windows [Version 10.0.20348.169]). I could see only supported version till windows server 2019. 

Can you please confirm whether its supported-on windows 2022, if yes from which version of Dokan its supported.

Thanks,
Subramanya N

**Comment by Liryna** [comment]
Hi @subramanya-n ,

I had no issue while testing Dokan on Windows Server 2022. https://github.com/dokan-dev/dokany/commit/ddc9eaef54f2327027f8eb845ec232f051b383bc

**Reply by subramanya-n** [reply]
Thank you for adding into certified list.
